dist/npm/README.md

```diff
@@ -1,289 +0,0 @@
-Typeahead, written using the React.js library.

```

Comment:
The whole dist folder is removed from source control with this PR.

It's been a while but I believe the readmes were identical.